brouter-core/src/main/java/btools/router/RoutingContext.java

```diff
@@ -261,14 +264,15 @@ public void cleanNogolist( List<OsmNodeNamed> waypoints )
     int n = nogopoints == null ? 0 : nogopoints.size();
     for( int i=0; i<n; i++ )
     {
-    	OsmNodeNamed nogo = nogopoints.get(i);
-    	cs[0] += nogo.ilon;
-    	cs[1] += nogo.ilat;
-    	cs[2] += (long) ( nogo.radius*111894.*10.);
+      OsmNodeNamed nogo = nogopoints.get(i);
+      cs[0] += nogo.ilon;
+      cs[1] += nogo.ilat;
+      // TODO[Phyks]
+      cs[2] += (long) ( nogo.radius*111894.*10.);

```

Comment:
Same question for the value of the constant. What is the `10` here for as well?

Reply:
factor 10 is arbitrary, just to have sub-meter precision in the checksum